pkg/lib/infra/middleware/cors.go

```diff
@@ -30,7 +30,7 @@ func (m *CORSMiddleware) Handle(next http.Handler) http.Handler {
 		allowedOrigins := m.Config.AllowedOrigins
 		allowedOrigins = append(allowedOrigins, m.CORSAllowedOrigins.List()...)
 		for _, oauthClient := range m.OAuthConfig.Clients {
-			allowedOrigins = append(allowedOrigins, oauthClient.RedirectURIHosts()...)
```

Comment:
The PR looks good with one minor comment.

We updated to add `RedirectURIs` directly into `allowedOrigins` directly without trimming the path here. I checked the updates in [originmatcher](https://github.com/iawaknahc/originmatcher) it should be fine. But it is better to add test case in `originmatcher` to indicate we accept origin with path. So when we want to update `originmatcher` in the future, we remember to keep this behavior.

Thought?

Reply:
Sure.